CI.hs

```diff
@@ -101,13 +103,28 @@ genVersionStr = \case
 
 parseOptions :: Opts.Parser Options
 parseOptions = Options
-    <$> Opts.switch
-        ( Opts.long "upload-to-bintray"
-       <> Opts.help "If specified, will try uploading the sdists to Bintray, using credentials in BINTRAY_BASIC_AUTH env var.")
-    <*> Opts.option readFlavor
+    <$> Opts.option readFlavor
         ( Opts.long "ghc-flavor"
        <> Opts.help "The ghc-flavor to test against"
         )
+    <*> Opts.strOption
+        ( Opts.long "da-base-sha"
```

Comment:
Maybe add a `ghc` in there and potentially `upstream` might be clearer than `base`.

Reply:
It's the base sha in the sense that it's the one we merge on to. Calling it upstream here would probably be confusing as we currently add our own fork as `upstream`, and this tag comes from `origin`.

Reply:
:+1:

Reply:
Renamed to `da-merge-base-sha`, hopefully that will be clearer.